File: Unidad_3/Java/Programa6_U3.java

package com.mycompany.Programa6_U3;
import java.util.Scanner;

public class Programa6_U3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese un número entero: ");
        int numero = scanner.nextInt();

        if (numero % 2 == 0) {
            System.out.println("Es un número par.");
        } else {
            System.out.println("Es un número impar.");
        }
    }
}

File: Unidad_3/C#/Programa6_U3.cs
using System;

class Programa6_U3
{
    static void Main()
    {
        Console.Write("Ingrese un número entero: ");
        int numero = int.Parse(Console.ReadLine());

        if (numero % 2 == 0)
        {
            Console.WriteLine("Es un número par.");
        }
        else
        {
            Console.WriteLine("Es un número impar.");
        }
    }
}
